travvy/src/components/Home.vue

```diff
@@ -12,7 +12,18 @@
     <br>
     <h2> Book your next trip!</h2>
     <!-- data binding destination, corresponding data object below -->
-    <label for="Destination">Select a Destination:</label>
+    <form>
+    <!-- this paragraph will only appear if there are any errors added to the errors array in the script below --> 
+    <p v-if="errors.length" class="error">
+      <b>Please enter the following missing information:</b>
+      <!-- this will bind all the errors together after the form is completed and return them to the user-->
+      <u1 id="checkForm" class="red"> 
+        <br>
+        <li v-for="error in errors" :key="error"> {{ error }}</li>
+        <!-- v-bind="error in errors"-->
+      </u1>
+    </p>
+    <label for="Destination">Select a Destination*:</label>
```

Comment:
Looks like you used some of the error handling code from sign up. Great to keep it consistent throughout the site and ensure when errors are returned to users, they are displayed in the same way throughout the site. Nice work!

Reply:
Thanks for figuring out how to implement it!